roles/webotsrobocup.role

```diff
@@ -0,0 +1,43 @@
+nuclear_role(
+  # FileWatcher, FileLogHandler and Signal Catcher Must Go First. KinematicsConfiguration usually goes after these and
+  # without it many roles do not run
+  extension::FileWatcher
+  support::SignalCatcher
+  support::logging::FileLogHandler
+  # This must come first as it emits config which many roles depend on (e.g. SensorFilter, WalkEngine)
+  motion::KinematicsConfiguration
+  # Support
+  support::configuration::GlobalConfig
+  network::NUClearNet
+  # support::NUsight
+  support::configuration::SoccerConfig
+  # input
+  input::GameController
+  # Simulator connection
+  input::SensorFilter
+  platform::Webots
+  # Vision
+  vision::VisualMesh
+  vision::GreenHorizonDetector
+  vision::GoalDetector
+  vision::BallDetector
+  # Localisation
+  localisation::BallLocalisation
+  localisation::RobotParticleLocalisation
+  # Behaviour
+  behaviour::Controller
+  behaviour::skills::Stand
+  behaviour::skills::FallingRelax
```

Comment:
We shouldn't have falling relax - the robot wont break itself anyway, and relaxing the joints currently just sends NaN to webots which is not happy at getting NaNs!